packages/DateRangePicker/src/DateRangePicker.js

```diff
@@ -0,0 +1,238 @@
+import React from "react";
+import PropTypes from "prop-types";
+import moment from "moment";
+
+import Popover from "@paprika/popover";
+import Calendar from "@paprika/calendar/lib/DateRangeCalendar";
+import { Kinds as CalendarKinds, START_DATE, END_DATE } from "@paprika/calendar/lib/tokens";
+import ArrowIcon from "@paprika/icon/lib/ArrowRightB";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import extractChildren from "@paprika/helpers/lib/extractChildren";
+import useDebounce from "@paprika/helpers/lib/hooks/useDebounce";
+import isElementContainsFocus from "@paprika/helpers/lib/dom/isElementContainsFocus";
+import { AlignTypes } from "@paprika/helpers/lib/customPropTypes";
+
+import createPropsCollector from "./createPropsCollector";
+import * as sc from "./DateRangePicker.styles";
+
+const propTypes = {
+  /** Selected start date in moment object */
+  startDate: PropTypes.instanceOf(moment),
+
+  /** Selected end date in moment object */
+  endDate: PropTypes.instanceOf(moment),
+
+  /** Callback to fire when user select start or end date */
+  onDatesChange: PropTypes.func.isRequired,
```

Comment:
I would suggest to uses `onChange` instead of onDateChange, onChange is what mostly all our component uses and easy to remember, if you feel strong about using your approach, then I would suggested switching to onChangeDates which followed our guidelines https://github.com/acl-services/paprika/wiki/Coding-Conventions#function-names